### PR TITLE
docs: add emojis to each section

### PR DESCRIPTION
**Pull request by Panquesito7**
### Describe the changes you've made

- Add emojis to each section.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

I don't think this applies here.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |
 
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/web3community/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,26 @@
 # Blockchain Developer Path
 
-Here you will learn how to become one of the Dapp Developers and collaborate with other people inside of web3community. We gather content about Blockchain developers, you don't need to watch it all but choose what content is best for you.
+Here you will learn how to become one of the Dapp Developers and collaborate with other people inside of Web3 Community. We gather content about Blockchain developers. You don't need to see all the resources, but choose what content is best for you.
```
Comment (KukretiShubham) [inline README.md:3]: I think this one needs no change. By web3community, we are mentioning our community. Not the general Web3 Community.

**Review by KukretiShubham — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shuklaritvik06) [reply README.md:3]: +1

Comment (KukretiShubham) [reply README.md:3]: web3community is the name of our community. And Web3, call it a tech stack or a new technology. Community focusing on Web3 is called a Web3 Community. We are also a Web3 Community, just our name is web3community. I know it's a bit confusing for you. Maybe @vinzvinci can explain in more simple words.

Comment (Panquesito7) [reply README.md]: Updated now. :)

**Review by krishguptadev — APPROVED** [review]

**Review by shuklaritvik06 — APPROVED** [review]

**Pull request merged**